Client/Frontend/Browser/EnhancedTrackingProtection/SlideoverPresentationController.swift

```diff
@@ -29,12 +27,7 @@ class SlideOverPresentationController: UIPresentationController {
     }
 
     override var frameOfPresentedViewInContainerView: CGRect {
-        var menuHeight: CGFloat
-        if globalETPStatus {
-            menuHeight = SlideOverUXConstants.ETPMenuHeightForGlobalOn
-        } else {
-            menuHeight = SlideOverUXConstants.ETPMenuHeightForGlobalOff
-        }
+        let menuHeight: CGFloat = presentedView!.systemLayoutSizeFitting(CGSize(width: presentedView!.bounds.width, height: 0.0), withHorizontalFittingPriority: .required, verticalFittingPriority: .defaultLow).height
```

Comment:
We'll soon be enabling a swiftlint rule that doesn't allow force unwrapping, so can we saftely unwrap this?

Reply:
Added an if-let block to do safe unwrap.